### ISSUE TITLE
Acquisition does not write to disk.

### ISSUE THREAD

**Issue by ksmith0** [ksmith0/PixieSuite#7]
After commit e734874938896a51a94a0ea5c60a7e15c05ae12e the acquisition no longer writes to disk. Starting via startvme and the trun does not produce a ldf as well as starting via the run command.

**Comment by ksmith0** [comment]
It would be nice to have an option to output the name of the current open ldf file. I expected ouf with no argument to do this, but I simply get the message "Set output file name to ''". It is unclear to me what the result is the resulting action of this command.

**Comment by ksmith0** [comment]
Acquisition does write to a file, but only after a spill occurs. Perhaps we should open the file immediately and provide the user with a message so that they see it working.

**Comment by cthornsb** [comment]
Output files are now opened immediately, not only after a spill has occured. This makes it more clear that the "trun" command has actually done something.  This feature was added in commit b858172.

**Comment by ksmith0** [comment]
This issue was moved to pixie16/PixieSuite2#29
